pyqtorch/primitive.py

```diff
@@ -302,6 +302,7 @@ def __init__(self, control: int | tuple[int, ...], target: int):
 
 
 OPS_PAULI = {X, Y, Z, I, N}
+OPS_PAULI_with_generator = {X, Y, Z, I}
```

Comment:
Just remove `N` from ops `OPS_PAULI`, it's more correct that way anyway, and it doesn't really affect much the tests since it is just a combination of `I` and `Z`.